apps/omg_watcher/test/fixtures.exs

```diff
@@ -144,7 +138,7 @@ defmodule OMG.Watcher.Fixtures do
         name: Watcher.Supervisor
       )
 
-    :ok = SQL.Sandbox.checkout(DB.Repo)
+    :ok = SQL.Sandbox.checkout(DB.Repo, ownership_timeout: 180_000)
```

Comment:
Why we need this - this fixture is used for quick controller's tests. There were no issues with them.